### PR TITLE
Footer: reorganize entries

### PR DESCRIPTION
**Pull request by erciccione**
Requires #1260 (I added a link to the 'Workgroup' page that doesn't exist yet).

I moved some stuff around, added and removed some entries, added a link to the Sitemap, removed the IRC Section and replaced with a more generic 'Reach Out' section.

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -9,29 +9,29 @@ <h3>{% t navigation.resources %}</h3>
...
+                                 <li><a class="white" href="https://mattermost.getmonero.org/">Mattermost</a></li>
+                                 <li><a class="white" href="https://telegram.me/monero">Telegram chat</a></li>
+
                             </ul>
                     </div>
                     <div class="col-sm-3 col-xs-6">
                         <h3>{% t navigation.community %}</h3>
                             <ul class="list-unstyled">
-                                 <li><a class="white" href="https://reddit.com/r/monero">Reddit</a></li>
+                                 <li><a class="white" href="{{ site.baseurl}}/community/merchants/">{% t titles.merchants %}</a></li>
```
Comment (SamsungGalaxyPlayer) [inline _includes/footer.html:31]: Why remove Reddit link?

File: _includes/footer.html [hunk 2]
```diff
@@ -9,29 +9,29 @@ <h3>{% t navigation.resources %}</h3>
...
-                                 <li><a class="white" href="irc://chat.freenode.net/#monero-markets">monero-markets</a></li>
-                                 <li><a class="white" href="irc://chat.freenode.net/#monero-pools">monero-pools</a></li>
-                                 <li><a class="white" href="irc://chat.freenode.net/#monero-community">monero-community</a></li>
-                                 <li><a class="white" href="irc://chat.freenode.net/#monero-translations">monero-translations</a></li>
-                                 <li><a class="white" href="irc://chat.freenode.net/#monero-hardware">monero-hardware</a></li>
+                                 <li><a class="white" href="{{ site.baseurl}}/community/team/">{% t titles.workgroups %}</a></li>
+                                 <li><a class="white" href="{{ site.baseurl}}/community/hangouts/">{% t titles.hangouts %}</a></li>
+                                 <li><a class="white" href="https://lists.getmonero.org/postorius/lists/monero-announce.lists.getmonero.org/">{% t footer.ml %}</a></li>
+                                 <li><a class="white" href="{{ site.baseurl_root }}/feed.xml">{% t footer.feed %}</a></li>
```
Comment (SamsungGalaxyPlayer) [inline _includes/footer.html:22]: Would this be better by the legal/source code/sitemap? Seems a little out of place under "Reach Out" to me

File: _includes/footer.html (hunk 1 above)
Comment (erciccione) [reply _includes/footer.html:30]: I added the Hangouts page in the Reach out colum and that includes all platforms where the community is present, reddit included.

File: _includes/footer.html (hunk 2 above)
Comment (erciccione) [reply _includes/footer.html]: Agreed.

**Comment by erciccione** [comment]
Rebased

**Review by selsta — APPROVED** [review]

**Pull request merged**